packages/react-menu/Spec.md

```diff
@@ -0,0 +1,63 @@
+# @fluentui/react-menu Spec
```

Comment:
Should be this file updated/deleted?

Reply:
I moved the spec here like I did with the larger draft PR